js/controller.js

```diff
@@ -403,6 +405,16 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.renderSkin();
     },
 
+    // NYM – enable closed captions when volume is off, or disable closed captions when volume is on
+    // for v4 players that default to displaying closed captions if available
+    nymOnVolumeChangedToggleClosedCaptions: function (event, newVolue) {
+      this.state.persistentSettings.closedCaptionOptions['enabled']
+      if ((newVolue <= 0 && !this.state.closedCaptionOptions.enabled)
+        || (newVolue > 0 && this.state.closedCaptionOptions.enabled)) {
+        this.toggleClosedCaptionEnabled();
+      }
+    },
```

Comment:
Updating to behavior that looks towards initial state vs toggle as discussed in our standup.